config/import-control.xml

```diff
@@ -169,13 +169,20 @@
     <allow pkg="javax.swing"/>
     <allow pkg="java.util" exact-match="true"/>
     <allow pkg="com.puppycrawl.tools.checkstyle.api" local-only="true"/>
-    <allow pkg="com.puppycrawl.tools.checkstyle.utils" local-only="true"/>
-    <allow class="com.puppycrawl.tools.checkstyle.JavaParser" local-only="true"/>
     <allow class="com.puppycrawl.tools.checkstyle.gui.MainFrameModel.ParseMode"
            local-only="true"/>
-    <allow class="com.puppycrawl.tools.checkstyle.JavadocDetailNodeParser" local-only="true"/>
     <allow class="com.google.common.collect.ImmutableList" local-only="true"/>
-    <allow class="java.nio.charset.StandardCharsets" local-only="true" />
+    <disallow pkg="com\.puppycrawl\.tools\.checkstyle\.(checks|ant|doclets|filters)" regex="true"/>
+
+    <unit name=".*(Model|Presentation)" regex="true">
+      <disallow pkg="java.awt.Graphics"/>
+      <disallow pkg="java.awt.Component"/>
+      <disallow pkg="javax\.swing\.J.*" regex="true"/>
+      <allow class="java.nio.charset.StandardCharsets" local-only="true" />
```

Comment:
what does "local-only" mean for "unit" ?
please remove all instances and lets update documentation that such parameter is not used for units as unit is most deep verification item of Check.

Reply:
This was a mistake to leave it in as this was a copy/paste.

`local-only` still has the same meaning here. Package must match exactly. File name matching is such that you can match a class that is in a sub-package.
There is nothing wrong with combining `file` and `local-only` in this way.